geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/ExecutionHandlerContext.java

```diff
@@ -257,8 +204,6 @@ public void channelInactive(ChannelHandlerContext ctx) {
         logger.debug("GeodeRedisServer-Connection closing with " + ctx.channel().remoteAddress());
       }
 
-      commandQueue.clear();
-      commandQueue.offer(TERMINATE_COMMAND);
```

Comment:
It looks like TERMINATE_COMMAND is no longer used. This class creates in a private static final so I think that static should also be removed